lexnlp/extract/pt/courts.py

```
@@ -0,0 +1,147 @@
+
+import os
+import re
+import warnings
+from collections.abc import Generator
+from typing import Any
+
+from lexnlp.extract.all_locales.languages import LANG_PT
+from lexnlp.extract.common.annotations.court_annotation import CourtAnnotation
+from lexnlp.extract.common.base_path import lexnlp_base_path
+from lexnlp.extract.common.universal_court_parser import ParserInitParams, UniversalCourtsParser
+from lexnlp.extract.en.dict_entities import (
```

Pylint (4.0.5)

[error] Cannot import 'lexnlp.extract.en.dict_entities' due to 'unexpected indent (lexnlp.extract.en.dict_entities, line 100)'

(E0001)